src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -184,12 +184,12 @@ public JedisSentinelPool(String masterName, Set<HostAndPort> sentinels,
   }
 
   @Override
-  public void destroy() {
+  public void close() {
```

Comment:
@sazzad16 let's keep the destroy method for backward you can just call close() on destory()